homeassistant/bootstrap.py

```diff
@@ -262,8 +290,7 @@ def enable_logging(hass, verbose=False, daemon=False, log_rotate_days=None):
                 }
             ))
         except ImportError:
-            _LOGGER.warning(
-                "Colorlog package not found, console coloring disabled")
```

Comment:
Installing by default sounds good. Colorless log is awful once you have seen the colored log.